edx_django_utils/management/README.md

```diff
@@ -0,0 +1,3 @@
+# Management Commands
```

Comment:
Thanks for adding this. 

1. This should be a `.rst` file. If you want more info on why or how, see https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-documentation.html#learning-rst
2. Could you add some text like the top-level README. For example, something like:
```
Django management commands useful to multiple services.

Note that some management commands may better belong in a more specific repository. A judgement call needs to be made as to whether code properly belongs here or not. Please review with the Architecture Team if you have any questions.
```
3. If using these commands requires adding to installed apps, please provide instructions here.